src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -27,10 +28,14 @@ export default class Checkbox extends React.PureComponent<Props> {
             onChange,
             children,
             className,
+            inactive,
         } = this.props;
         const checkboxClass = classNames(
             checkboxStyles.checkbox,
-            checkboxStyles[skin],
+            {
+                [checkboxStyles.inactive]: inactive,
+                [checkboxStyles[skin]]: !inactive,
```

Comment:
Any reason not to set the skin when the button is not active? Maybe there will even be different inactive styles for each skin at some point.

Reply:
currently the same css attributes are set differently. so there is currently no difference - but we can set both classes.